terraform/modules/prober/variables.tf

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+variable "project_id" {
+  type        = string
+  description = "The GCP project to host the prober."
+}
+
+variable "project_number" {
```

Comment:
Should we get the project number in terraform based on the project_id instead of asking users for both project id and project number?

Reply:
Removed this part, turns out we can use project_id instead of project_number.